app/controllers/concerns/rendering.rb

```diff
@@ -11,14 +11,24 @@ def render_json(model, **args)
       render({ json: serializer.new(model, opts) }.merge(args))
     end
 
-    def render_error(model, **args)
+    def render_error(models, **args)
```

Comment:
Observation: This could accept a string or list of strings of error messages.

... so we would prevent rendering errors like
```
render json: { errors: error }, status: :forbidden 
```

Rationale: If the the method of rendering error changes, then all places have to change with it.

Reply:
We could add that, and this method could use that method, I guess. For this PR, though, I don't want to update all the places where we do manual rendered error messages.